ql/src/test/results/clientpositive/llap/acid_insert_overwrite_update.q.out

```diff
@@ -243,16 +247,28 @@ POSTHOOK: Input: default@orc_test_txn@year=2018
 LOOKS OKAY
 PREHOOK: query: alter table orc_test_txn partition(year='2016') compact 'major'
 PREHOOK: type: ALTERTABLE_COMPACT
+PREHOOK: Input: default@orc_test_txn
+PREHOOK: Output: default@orc_test_txn
 POSTHOOK: query: alter table orc_test_txn partition(year='2016') compact 'major'
 POSTHOOK: type: ALTERTABLE_COMPACT
+POSTHOOK: Input: default@orc_test_txn
+POSTHOOK: Output: default@orc_test_txn
 PREHOOK: query: alter table orc_test_txn partition(year='2017') compact 'major'
 PREHOOK: type: ALTERTABLE_COMPACT
+PREHOOK: Input: default@orc_test_txn
+PREHOOK: Output: default@orc_test_txn
 POSTHOOK: query: alter table orc_test_txn partition(year='2017') compact 'major'
 POSTHOOK: type: ALTERTABLE_COMPACT
+POSTHOOK: Input: default@orc_test_txn
+POSTHOOK: Output: default@orc_test_txn
 PREHOOK: query: alter table orc_test_txn partition(year='2018') compact 'major'
 PREHOOK: type: ALTERTABLE_COMPACT
+PREHOOK: Input: default@orc_test_txn
```

Comment:
why does partition 'year=2018' not present in the Input/Output?

Reply:
Ack